example-rule-library/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+ Copyright 2018-2021 Crown Copyright
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+# <img src="../logos/logo.svg" width="180">
+
+## A Tool for Complex and Scalable Data Access Policy Enforcement
+
+# Example Rule library
+Contains the Rules and supporting code used in the example.
+The full explanation of how this library is used in context to the example is described in the [Example Library](./example-library/README.md).
```

Comment:
I think it would be better to have the Rules stuff in here rather than in the example lib readme

Reply:
Yes, the description does not belong here.  What I think would work better is a bit in the example-rule-library and some of it in the root readme under a new heading Overview of the Example.